EventStream/data/dataset_polars.py

```
@@ -1612,33 +1645,30 @@ def _summarize_dynamic_measurements(
                     key_col.is_not_null()
                     .sum()
                     .cast(count_type)
-                    .map_alias(lambda c: f"dynamic/{m}/{c.replace(key_prefix, '')}/count"),
+                    .name.map(lambda c: f"dynamic/{m}/{c.replace(key_prefix, '')}/count"),
```

Ruff (0.8.2)

Function definition does not bind loop variable `m`

(B023)